pkg/users/api_key.go

```diff
@@ -6,8 +6,8 @@ import (
 	"github.com/OctopusDeploy/go-octopusdeploy/v2/pkg/resources"
 )
 
-// APIKey represents an API key.
-type APIKey struct {
+// CreateAPIKey represents an API key when returned from creating a new API key (POST)
```

Comment:
Looks like this is used as both the request and the response

```suggestion
// CreateAPIKey represents an API key when creating a new API key (POST)
```
which makes `CreateAPIKey` kind of a weird name but I don't have any good suggestions so feel free to leave it.

Reply:
will keep the name for now as I can't think of anything else 😅